tests/004-test-cleanup

```diff
@@ -0,0 +1,82 @@
+source $SRCDIR/libtest.sh
+
+cleanup() {
+  local vg_name=$1
+  local devs=$2
+  $DSSBIN internal-only-for-atomic-reset >> $LOGS 2>&1
+  test -e /etc/sysconfig/docker-storage && echo "ERROR: $0: Failed." && exit 1
+  clean_config_files
```

Comment:
Tests are not supposed to exit on their own. They are supposed to do all the cleanup and return success/failure status. Parent does all the logging w.r.t whether test passed or failed.

So once we have called "dss reset", we just need to make sure lvm thin pool went away and /etc/sysconfig/docker-storage went away and determine test status based on that.

And then call cleanup() to cleanup everything (even if test failed). This will allow us to continue running other tests even if some tests failed and these tests will require a clean environment to run.